plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergSplit.java

```diff
@@ -48,7 +49,8 @@
 
     @JsonCreator
     public IcebergSplit(
-            @JsonProperty("path") String path,
+            @JsonProperty("displayPath") String displayPath,
```

Comment:
Having both `displayPath` and `path` is rather confusing.

```
                task.file().path().toString(), //displayPath
                hadoopPath(task.file().path().toString()), //path
```

They are both built from the same string. 
Keep only `path` corresponding to `task.file().path().toString()` and apply `hadoopPath` on the path within `io.trino.plugin.iceberg.IcebergPageSourceProvider#createPageSource` (the only place where `IcebergSplit#getPath()` is actually used.